_posts/2016-12-03-Mermaid.md

```diff
@@ -43,18 +51,20 @@ sequenceDiagram
 
 ### Flow
 
-<div class="mermaid">
+```mermaid
```

Comment:
`<div class="mermaid">` is still supported, but I prefer the more markdown integrated way.